src/pages/Notification/Notifications.tsx

```diff
@@ -1,65 +1,59 @@
 /* eslint-disable jsx-a11y/no-static-element-interactions */
 /* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
Is there a reason for these rules to be disabled?

Reply:
Caique left it there. idk to be honest

Reply:
fixed